pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, openssl, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "open-isns-${version}";
+  version = "0.95";
+
+  src = fetchFromGitHub {
+    owner = "gonzoleeman";
+    repo = "open-isns";
+    rev = "v${version}";
+    sha256 = "1c2x3yf9806gbjsw4xi805rfhyxk353a3whqvpccz8dwas6jajwh";
+  };
+
+  propagatedBuildInputs = [ openssl ];
+  outputs = ["out" "lib" ];
+  outputInclude = "lib";
+
+  installFlags = "etcdir=$out/etc vardir=$out/var/lib/isns";
```

Comment:
Good catch! I failed to copy your suggestion verbatim and also didn't spot the broken output paths in build output. Fixed.

vardir isn't customized through any prefix given to ./configure, which is working as intended as we don't want to daemons to write there. Starting isnsd will make it write its pid into /var/run.